src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/TraceReporter.kt

```diff
@@ -380,6 +379,57 @@ internal fun constructTraceGraph(
     return traceGraphNodesSections.map { it.first() }
 }
 
+
+/**
+ * Merges fun$default(...) calls
```

Comment:
Could you please make the documentation a bit more general?
In the future, we might want to modify this function to also compress some other patterns.

Reply:
Sure thing